### PR TITLE
WFLY-18473: helloworld-jms Quickstart Common Enhancements CY2023Q3.

### PR DESCRIPTION
**Pull request by ehsavoie**
* Using integration-testing as a way to execute the client
* provisionned server is ready to handle the client

Issue: https://issues.redhat.com/browse/WFLY-18473

### CODE REVIEW

File: helloworld-jms/README.adoc [hunk 1]
```diff
@@ -117,9 +102,9 @@ timestamp org.jboss.as.quickstarts.jms.HelloWorldJMSClient main
 INFO: Received message with content Hello, World!
 ----
 
-== Optional Properties
+=== Optional Properties
```
Comment (emmartins) [inline helloworld-jms/README.adoc:105]: Personally I would get ridden of those (here and in the pom.xml) and hardcode them in the test class, like other QS are doing. The only non constant would be the java.naming.provider.url but you could derivate the value for that one from the server.host param used in tests

File: helloworld-jms/src/test/resources/helloworld-jms-logging.properties [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#
```
Comment (emmartins) [inline helloworld-jms/src/test/resources/helloworld-jms-logging.properties:1]: do we really need logging properties on a helloworld? This quickstart should be as simple as possible, focusing on JMS only)

File: helloworld-jms/src/main/resources/helloworld-jms-logging.properties [hunk 3]
```diff
@@ -0,0 +1,39 @@
+#
```
Comment (emmartins) [inline helloworld-jms/src/main/resources/helloworld-jms-logging.properties:1]: do we really need logging properties on a helloworld? This quickstart should be as simple as possible, focusing on JMS only)

File: helloworld-jms/README.adoc [hunk 4]
```diff
@@ -167,32 +152,4 @@ This script removes the `test` queue from the `messaging` subsystem in the serve
...
-// Product Release content only
-//*************************************************
-ifdef::ProductRelease[]
-
-// Quickstart not compatible with OpenShift
-include::../shared-doc/openshift-incompatibility.adoc[leveloffset=+1]
-
-endif::[]
\ No newline at end of file
+include::../shared-doc/openshift-incompatibility.adoc[leveloffset=+1]
```
Comment (emmartins) [inline helloworld-jms/README.adoc:155]: Before this please include the provisioning instructions:

```
// Build and run sections for other environments/builds
ifndef::ProductRelease,EAPXPRelease[]
include::../shared-doc/build-and-run-the-quickstart-with-provisioned-server.adoc[leveloffset=+1]
endif::[]
```

**Comment by emmartins** [comment]
In generally I think this quickstart could be much simplified, by removing the optional properties (which are not used anywhere in the user flow), and also the logging properties. This would mean everything non standard on it would focus on JMS only. Those changes are optional of course, just suggestions, the only mandatory change needed is to include the provisioning instructions.

**Review by emmartins — CHANGES_REQUESTED** [review]
In generally I think this quickstart could be much simplified, by removing the optional properties (which are not used anywhere in the user flow), and also the logging properties. This would mean everything non standard on it would focus on JMS only. Those changes are optional of course, just suggestions, the only *mandatory* change needed is to include the provisioning instructions.

File: helloworld-jms/README.adoc (hunk 1 above)
Comment (ehsavoie) [reply helloworld-jms/README.adoc:105]: I put them here for retro compatibility

**Pull request merged**